### PR TITLE
Added a note about the /v2/module endpoint for RS

### PR DESCRIPTION
**Pull request by dallastexas92**
There is no explanation that the v2/module endpoint only exists for RS v6.0.12 and above. I added clarification to this when adding a module via the REST API

### CODE REVIEW

**Comment by dallastexas92** [comment]
@MeirShpilraien can you please take a look at this PR when you get a moment?

File: content/modules/add-module-to-cluster.md
Comment (kaitlynmichael) [inline content/modules/add-module-to-cluster.md:37]:
```suggestion
**Note:** The /v2/modules endpoint only exists for Redis Enterprise Software v6.0.12 and above. If you are on an older version, you must perform a manual installation of the module and its dependencies 
```

**Review by kaitlynmichael — APPROVED** [review]
Just a few wording changes from me. If this is accurate and approved, I can merge it.

File: content/modules/add-module-to-cluster.md
Comment (dallastexas92) [reply content/modules/add-module-to-cluster.md:37]: Thanks @kaitlynmichael ! Still waiting on a response from @MeirShpilraien before merging

**Comment by kaitlynmichael** [comment]
@dallastexas92 are we still waiting on more approvals before we merge this?

**Comment by MeirShpilraien** [comment]
@dallastexas92 looks ok to me...

**Comment by dallastexas92** [comment]
@kaitlynmichael all set - thank you!!

**Pull request merged**
